--- snapshot-47.0.1/src/meson.build.orig	2024-09-15 20:59:11.000000000 +0200
+++ snapshot-47.0.1/src/meson.build	2024-10-05 08:20:02.018366043 +0200
@@ -25,6 +25,7 @@ run_command(
 )
 
 cargo_options = [ '--manifest-path', meson.project_source_root() / 'Cargo.toml' ]
+cargo_options += [ '--target', 'x86_64-unknown-linux-gnux32' ]
 cargo_options += [ '--target-dir', meson.project_build_root() / 'src' ]
 
 if is_devel
@@ -61,7 +62,7 @@ cargo_build = custom_target(
     cargo, 'build',
     cargo_options,
     '&&',
-    'cp', 'src' / rust_target / meson.project_name(), '@OUTPUT@',
+    'cp', 'src'/ 'x86_64-unknown-linux-gnux32' / rust_target / meson.project_name(), '@OUTPUT@',
   ]
 )
 
