--- snapshot-50.0/src/meson.build.orig	2026-03-11 20:45:59.000000000 +0100
+++ snapshot-50.0/src/meson.build	2026-10-04 11:22:52.490195667 +0200
@@ -31,6 +31,7 @@ if gtk4_x11.found()
 endif
 
 cargo_build_options += cargo_release_options
+cargo_build_options += [ '--target', 'x86_64-unknown-linux-gnux32' ]
 
 cargo_build = custom_target(
   'cargo-build',
@@ -47,6 +48,6 @@ cargo_build = custom_target(
     'build',
     cargo_build_options,
     '&&',
-    'cp', cargo_target / rust_target / meson.project_name(), '@OUTPUT@',
+    'cp', cargo_target / 'x86_64-unknown-linux-gnux32' / rust_target / meson.project_name(), '@OUTPUT@',
   ]
 )
